lib/rest2.js

```diff
@@ -114,6 +115,11 @@ class RESTv2 {
     })
   }
 
+  _takeNotificationInfo (data) {
```

Comment:
Since this method doesn't access `this`, can you extract it as a utility function?